backend/backend/production_settings.py

```diff
@@ -12,29 +12,23 @@
 
 import os
 from pathlib import Path
+from .settings import *
+import sys
+import mimetypes
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+mimetypes.add_type("text/css", ".css", True)
```

Comment:
Why id you need to write this line? What's the issue with the default?

Reply:
For whatever reason, with the new setup, the CSS static file were not receiving the correct mime types and django wasn't applying them even though it was fetching them. I'm not actually sure why.